moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h

```diff
@@ -344,6 +344,13 @@ class PlanningSceneMonitor : private boost::noncopyable
   /** @brief This function is called every time there is a change to the planning scene */
   void triggerSceneUpdateEvent(SceneUpdateType update_type);
 
+  /** \brief Wait for robot state to become more recent than t.
+   *
+   * If there is no state monitor active, there will be no scene updates.
+   * Hence, you can specify a timeout to wait for those updates. Default is 1s.
+   */
+  bool waitForCurrentRobotState(const ros::Time &t, double wait_time = 1.);
```

Comment:
As explained by @v4hn, we agreed to correctly rename the old function waitForCurrentRobotState -> waitForCompleteRobotState to reflect its actual semantics.
The new function waitForCurrentRobotState actually does what its name states: waiting for an update of the current robot state.